data/helpers.d/setting

```diff
@@ -230,11 +232,24 @@ ynh_webpath_register () {
 
 # Create a new permission for the app
 #
+# example: ynh_permission_create --permission admin --urls /admin
+#
 # usage: ynh_permission_create --permission "permission" [--urls "url" ["url" ...]]
 # | arg: permission - the name for the permission (by default a permission named "main" already exist)
 # | arg: urls - (optional) a list of FULL urls for the permission (e.g. domain.tld/apps/admin)
+# | arg: urls - (optional) a list of urls to specify for the permission.
+#
+#  Urls are assumed to be relative to the app domain/path if they start with '/'.
+#  For example:
+#     /                             -> domain.tld/app
+#     /admin                        -> domain.tld/app/admin
+#     domain.tld/app/api            -> domain.tld/app/api
```

Comment:
Should that be "API"?

Reply:
uuuuuh idk, it's really just an example to explain what's going on, the case doesn't really matter ...